test/unit/test_loss_image.py

```diff
@@ -69,28 +69,24 @@ def test_get_config(self):
 @pytest.mark.parametrize("name", ["gaussian", "triangular", "rectangular"])
 def test_kernel_fn(name):
     kernel_size = 3
-    input_channel = 5
     kernel_fn = image.LocalNormalizedCrossCorrelation.kernel_fn_dict[name]
-    filters, kernel_vol = kernel_fn(kernel_size, input_channel)
+    filters = kernel_fn(kernel_size)
     assert filters.shape == (
         kernel_size,
-        kernel_size,
-        kernel_size,
-        input_channel,
+        1,
         1,
     )
-    assert kernel_vol.shape == ()
 
 
 class TestLocalNormalizedCrossCorrelation:
     @pytest.mark.parametrize(
         "y_true,y_pred,shape,kernel_type,expected",
         [
-            (0.6, 0.3, (3, 3, 3, 3), "rectangular", 1.0),
-            (0.6, 0.3, (3, 3, 3, 3, 3), "rectangular", 1.0),
-            (0.0, 1.0, (3, 3, 3, 3, 3), "rectangular", 1.0),
-            (0.6, 0.3, (3, 3, 3, 3, 3), "gaussian", 1.0),
-            (0.6, 0.3, (3, 3, 3, 3, 3), "triangular", 1.0),
+            (0.6, 0.3, (12, 12, 12, 12), "rectangular", 1.0),
```

Comment:
any particular reason we need them to be big? i reduce the size to make unit test faster

Reply:
I think it needs to be larger than the kernel size (not specified here, 9 by default). You could input kernel size as a parameter as well.